pkg/controllers/jobset_controller.go

```diff
@@ -207,18 +210,18 @@ func (r *JobSetReconciler) createJobs(ctx context.Context, js *jobset.JobSet, ow
 
 		// Create headless service if specified for this job.
 		if rjob.Network.EnableDNSHostnames != nil && *rjob.Network.EnableDNSHostnames {
-			klog.Infof("creating headless service: %s", job.Name)
+			log.Info("creating headless service", "service", job.Name)
```

Comment:
v(2)

Reply:
I would also move it after the call and say: "successfully created headless service"